app-ui-catalog/src/main/java/app/k9mail/ui/catalog/ui/organism/items/AppBarItems.kt

```diff
@@ -15,13 +15,31 @@ import app.k9mail.ui.catalog.ui.common.list.sectionHeaderItem
 import app.k9mail.ui.catalog.ui.common.list.sectionSubtitleItem
 
 fun LazyGridScope.appBarItems() {
+    topAppBarItems()
+    subtitleTopAppBarItems()
+}
+
+private fun LazyGridScope.topAppBarItems() {
     sectionHeaderItem(text = "TopAppBar")
     sectionSubtitleItem(text = "With menu icon")
     fullSpanItem {
         ItemOutlinedView {
-            TopAppBarItem(
+            TopAppBar(
                 title = "Title",
-                actionIcon = Icons.Outlined.Info,
+                actions = {
+                    ButtonIcon(
+                        onClick = {},
+                        imageVector = Icons.Outlined.Info,
+                    )
+                    ButtonIcon(
+                        onClick = {},
+                        imageVector = Icons.Outlined.Check,
+                    )
+                    ButtonIcon(
+                        onClick = {},
+                        imageVector = Icons.Outlined.Visibility,
+                    )
+                },
```

Comment:
Help me understand this a bit more. It looks to me like we're creating a template where the onClick handler can be added later? I see this pattern multiple times in the file, maybe there is an opportunity to DRY it up?

Reply:
The `ButtonIcon` requires a `onClick` handler, but as we're not interested in any action when clicking the button, it is set to `{}`. This could be dryed by having a new button that already defines `onClick = {}`.

The catalog is solely there to demonstrate the different design system components and we didn't put too much effort into ironing out repeated content.

Reply:
Added a DemoActionButton